docs/installation/htcondor-ce.md

```diff
@@ -159,6 +159,22 @@ as the following HTCondor-CE configuration:
 
             globus_mapping /usr/lib64/libgsi_pep_callout.so argus_pep_callout
 
+#### (Optional) Use grid certificates for SSL authentication ####
+
+By default, HTCondor-CE uses system certificates for authenticating SSL connections and expects a host certificate and
+key in `/etc/grid-security/condor/hostcert.pem` and `/etc/grid-security/condor/hostkey.pem`, respectively.
+If you or your clients are using [IGTF-accredited certificates](https://dl.igtf.net/distribution/igtf/current/accredited/accredited.in),
```

Comment:
IGTF certificates aren't really the deciding factor, are they?  It's more whether they use the system trust roots or the traditional grid location for trust roots.  How about:

> By default, HTCondor-CE uses the default system locations to locate certificate authorities (CAs) and host certificate when authenticating SSL connections.  If you would prefer to to use the traditional "grid location" for CAs and the host certificate, set the following configuration in `/etc/condor-ce/config.d/01-ce-auth.conf`:

?

Reply:
How's it look now?